src/ui/components/ExperimentBrowserPage.tsx

```diff
@@ -0,0 +1,113 @@
+import { FC, useCallback, useEffect, useState } from "react";
+
+const PROD_URL =
+  "https://experimenter.services.mozilla.com/api/v7/experiments/";
+const STAGE_URL =
+  "https://stage.experimenter.nonprod.dataops.mozgcp.net/api/v6/experiments/";
+
+interface Experiment {
```

Comment:
You should use the `NimbusExperiment` type from the (`@mozilla/nimbus-schemas`](https://www.npmjs.com/package/@mozilla/nimbus-schemas) package on npm